esmvalcore/cmor/_fixes/fix.py

```diff
@@ -146,9 +162,9 @@ def get_fixes(project, dataset, variable):
 
             classes = inspect.getmembers(fixes_module, inspect.isclass)
             classes = dict((name.lower(), value) for name, value in classes)
-            for fix_name in ('allvars', variable):
+            for fix_name in (short_name, 'allvars'):
```

Comment:
I'm not sure if it's a good idea to change the order here, because you will have to check that all fixes that are already implemented still work. @jvegasbsc Do you have an opinion on this?